pkg/debugger/policy_test.go

```diff
@@ -4,49 +4,46 @@ import (
 	"net/http/httptest"
 	"testing"
 
-	"github.com/golang/mock/gomock"
+	gomock "github.com/golang/mock/gomock"
+	"github.com/openservicemesh/osm/pkg/compute/kube"
+	"github.com/openservicemesh/osm/pkg/k8s"
```

Comment:
this linter error is annoying 😃 fyi you can configure your IDE to do this automatically which has solved having to update this everytime

Reply:
Actually probably going to get rid of this file altogether due to upcoming change I'm about to make, but will keep this in mind!